x/ccv/provider/types/keys.go

```diff
@@ -31,324 +32,499 @@ const (
+		SlashLogKeyName: 26,
+
+		// ConsumerRewardDenomsKey is the key for storing a list of consumer reward denoms
+		ConsumerRewardDenomsKeyName: 27,
+
+		// VSCMaturedHandledThisBlockKey is the key for storing the number of vsc matured packets
+		// handled in the current block
+		VSCMaturedHandledThisBlockKeyName: 28,
+
+		// EquivocationEvidenceMinHeightKey is the key for storing the mapping from consumer chain IDs
+		// to the minimum height of a valid consumer equivocation evidence
+		EquivocationEvidenceMinHeightKeyName: 29,
+
+		// ProposedConsumerChainKey is the key for storing the consumer chainId in consumerAddition gov proposal submitted before voting finishes
+		ProposedConsumerChainKeyName: 30,
+
+		// ConsumerValidatorKey is the key for storing for each consumer chain all the consumer
+		// validators in this epoch that are validating the consumer chain
+		ConsumerValidatorKeyName: 31,
+
+		// OptedInKey is the key for storing whether a validator is opted in to validate on a consumer chain
+		OptedInKeyName: 32,
+
+		// TopNKey is the key for storing the mapping from a consumer chain to the N value of this chain,
+		// that corresponds to the N% of the top validators that have to validate this consumer chain
+		TopNKeyName: 33,
+
+		// ValidatorsPowerCapKey is the key for  storing the mapping from a consumer chain to the power-cap value of this chain,
+		// that corresponds to p% such that no validator can have more than p% of the voting power on the consumer chain.
+		// Operates on a best-effort basis.
+		ValidatorsPowerCapKeyName: 34,
+
+		// ValidatorSetCapKey is the key for storing the mapping from a consumer chain to the validator-set cap value
+		// of this chain.
+		ValidatorSetCapKeyName: 35,
+
+		// AllowlistKey is the key for storing the mapping from a consumer chain to the set of validators that are
+		// allowlisted.
+		AllowlistKeyName: 36,
+
+		// DenylistKey is the key for storing the mapping from a consumer chain to the set of validators that are
+		// denylisted.
+		DenylistKeyName: 37,
+
+		// ConsumerRewardsAllocationKey is the key for storing for each consumer the ICS rewards
+		// allocated to the consumer rewards pool
+		ConsumerRewardsAllocationKeyName: 38,
+
+		// ConsumerCommissionRateKey is the key for storing the commission rate
+		// per validator per consumer chain
+		ConsumerCommissionRateKeyName: 39,
+
+		// MinimumPowerInTopNKey is the key for storing the
+		// minimum power required to be in the top N per consumer chain.
+		MinimumPowerInTopNKeyName: 40,
+
+		// NOTE: DO NOT ADD NEW BYTE PREFIXES HERE WITHOUT ADDING THEM TO TestPreserveBytePrefix() IN keys_test.go
+	}
+}
+
+// mustGetKeyPrefix returns the key prefix for a given key.
+// It panics if there is not byte prefix for the index.
+func mustGetKeyPrefix(key string) byte {
+	keyPrefixes := getKeyPrefixes()
+	if prefix, found := keyPrefixes[key]; !found {
+		panic(fmt.Sprintf("could not find key prefix for index %s", key))
+	} else {
```

Comment:
very nit: `else` statement is unnecessary here.

Reply:
without the else I need to define `prefix` outside the if.